### ISSUE TITLE
stats resources werken niet meer

### ISSUE THREAD

**Issue by wires** [erfgoed-en-locatie/histograph#61]
alles is `[]`
https://api.histograph.io//stats/queries/pits-and-relations-per-dataset

**Comment by wires** [comment]
```
Executing all Cypher queries...
/home/histograph/api/node_modules/histograph-stats/queries.js:24
    return db.cypher({
              ^
TypeError: undefined is not a function
    at /home/histograph/api/node_modules/histograph-stats/queries.js:24:15
    at Stream._generator (/home/histograph/api/node_modules/highland/lib/index.js:3994:15)
    at Stream._runGenerator (/home/histograph/api/node_modules/highland/lib/index.js:949:10)
    at Stream.resume (/home/histograph/api/node_modules/highland/lib/index.js:811:22)
    at Stream._checkBackPressure (/home/histograph/api/node_modules/highland/lib/index.js:713:17)
    at Stream.resume (/home/histograph/api/node_modules/highland/lib/index.js:806:29)
    at Stream.pull (/home/histograph/api/node_modules/highland/lib/index.js:1176:7)
    at Stream._generator (/home/histograph/api/node_modules/highland/lib/index.js:2782:14)
    at Stream._runGenerator (/home/histograph/api/node_modules/highland/lib/index.js:949:10)
    at Stream.resume (/home/histograph/api/node_modules/highland/lib/index.js:811:22)
    at Stream._generator_next (/home/histograph/api/node_modules/highland/lib/index.js:513:22)
    at /home/histograph/api/node_modules/highland/lib/index.js:2803:28
    at /home/histograph/api/node_modules/highland/lib/index.js:1173:9
    at Stream.s._send (/home/histograph/api/node_modules/highland/lib/index.js:1139:9)
    at Stream.write (/home/histograph/api/node_modules/highland/lib/index.js:1220:18)
    at Stream._send (/home/histograph/api/node_modules/highland/lib/index.js:657:32)
```

**Comment by bertspaan** [comment]
Fixed in 79e0af519b99b8e137a3951548a6f247f426af2a  in histograph/stats
